secator/cli.py

```
@@ -998,10 +998,25 @@ def _apply_format(results, fmt):
 		dict: Results dict with items replaced by formatted strings (only matching types kept).
 	"""
+	fmt = fmt.strip()
+
+	# Auto-detect format file: if fmt looks like a path and the file exists, load it.
+	if len(fmt) < 255:
+		try:
+			p = Path(fmt)
+			if p.is_file():
+				fmt = p.read_text()
+		except Exception:
+			pass
```

Ruff (0.15.12)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)